tst/regression/CMakeLists.txt

```diff
@@ -39,10 +39,11 @@
 
 # Performance regression test
 list(APPEND TEST_DIRS advection_performance)
-list(APPEND TEST_PROCS "1")
+list(APPEND TEST_PROCS ${NUM_MPI_PROC_TESTING})
```

Comment:
Wait... is `NUM_MPI_PROC_TESTING` now automatically set to 1 for the non-mpi tests?

Reply:
NUM_MPI_PROC_TESTING is not used for the serial tests.